README.md

```diff
@@ -4,22 +4,30 @@
 
 # 📘️ swagger-to-ts
 
-Convert [OpenAPI v2][openapi2] schemas to TypeScript interfaces using Node.js. It can handle large
-definition files within milliseconds because it neither validates nor parses; it only transforms the
-bare minimum of what it needs to.
+🚀 Convert [OpenAPI v2][openapi2] schemas to TypeScript interfaces using Node.js. 
 
-💅 Prettifies output with [Prettier][prettier].
+💅 The output is prettified with [Prettier][prettier].
+
+👉 Works for both local and remote resources (filesystem and http).
 
 To compare actual generated output, see the [example](./example) folder.
 
+(**swagger-to-ts** can handle large definition files within milliseconds because it neither validates nor parses; it only transforms the bare minimum of what it needs to.)
+
 ## Usage
 
 ### CLI
 
+#### Reading specs from file system
+
 ```bash
 npx @manifoldco/swagger-to-ts schema.yaml --output schema.d.ts
+```
+
+#### Reading specs from remote resource
 
-# 🚀 schema.yaml -> schema.d.ts [500ms]
+```bash
+npx @manifoldco/swagger-to-ts http://myapi.com/schema --output schema.d.ts
```

Comment:
What if we used a real endpoint here, like `npx @manifoldco/swagger-to-ts https://petstore.swagger.io/v2/swagger.json --output petstore.d.ts`? It’d let people try it themselves and see it work.

Reply:
Sure - I'm gonna update it :)